### PR TITLE
Workspace - charts

### PR DESCRIPTION
**Pull request by Piotrek605**

### CODE REVIEW

**Comment by Zodiase** [comment]
Please fix all linting errors. For example, using `ifvisible.js` without saving it as a dependency of the project.

**Comment by Piotrek605** [comment]
Description: added a pop-up window that appears after selecting a point on the map. The window contains charts displaying data for the selected point and sliders for choosing the year range. The window minimizes when the main window becomes inactive.

**Comment by Zodiase** [comment]
When running the app, I see this error in console:

image

Also the pop-up is not restored when the parent window becomes visible again.

File: meteor-app/imports/ui/pages/workspace/component.js [hunk 1]
```diff
@@ -1,6 +1,65 @@
...
+  theWindow._minimizeRestore = {
+    width: theWindow.outerWidth,
+    height: theWindow.outerHeight,
+    x: theWindow.screenX,
+    y: theWindow.screenY,
+  };
+}
+
+// Minimize the child window when the parent window becomes inactive
+ifvisible.on('blur', function () {
```
Comment (Zodiase) [inline meteor-app/imports/ui/pages/workspace/component.js]: These bindings should not happen globally. They should take effect **only** when this component is rendered, and everything should be properly cleaned up when this component exists.

**Comment by Piotrek605** [comment]
Try restarting meteor. We decided to give up on restoring the popup window because it’s difficult to maintain its dimensions and position.

**Comment by Zodiase** [comment]
Added a new commit for fixing the mis-placed `welcomeWindowClosed` state property. Please pull before making changes.

**Comment by Piotrek605** [comment]
Now the window should be restoring correctly, both when invoked manually and when the main window is restored.

**Pull request merged**